src/DashboardHub/Bundle/DevelopmentBundle/DependencyInjection/DashboardHubDevelopmentExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace DashboardHub\Bundle\DevelopmentBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\DependencyInjection\Loader;
+
+/**
+ * This is the class that loads and manages your bundle configuration
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
+ */
+class DashboardHubDevelopmentExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('services.yml');
```

Comment:
There is some stuff in the latest services.yml file...

services:
    mockdashboardhub_user_provider:
        class: DashboardHub\Bundle\DevelopmentBundle\Security\User\DashboardHubUserProvider